src/sentry/utils/sdk.py

```diff
@@ -69,6 +69,7 @@
     "sentry.tasks.store.process_event": settings.SENTRY_PROCESS_EVENT_APM_SAMPLING,
     "sentry.tasks.store.process_event_from_reprocessing": settings.SENTRY_PROCESS_EVENT_APM_SAMPLING,
     "sentry.tasks.assemble.assemble_dif": 0.1,
+    "sentry.tasks.app_store_connect.dsym_download": 1,
```

Comment:
@beezz is this, err, reasonable?

Reply:
So you want transactions for each run. How often will this run? I assume low frequency in that case it's ok.

Reply:
Currently this feature is in select people only, i.e. native team + mobile team.  Next week it should be in EA, no idea how fast adoption will go really.